rick.py

```diff
@@ -640,8 +640,8 @@ def stacked_chart(data_in, xlab, lab1, lab2, **kwargs):
             data_yoy['percent'] = (data['values2']-data['values1'])*100/data['values1']
             j=0.15
             for index, row in data_yoy.iterrows():
-                ax.annotate(('+' if row['percent'] > 0 else '')+str(format(int(round(row['percent'],0)), ','))+'%', 
-                            xy=(max(row[['values1', 'values2']]) + (0.12 if row['values2'] < 0.1*upper else 0.03)*upper, j), color = 'k', fontname = font.normal, fontsize=10)
+                ax.annotate('+'+str(format(int(round(row['percent'],0)), ','))+'%',
+                            xy=(max(row[['values1', 'values2']]) + 0.03*upper, j), color = 'k', fontname = font.normal, fontsize=10)

```

Comment:
found in vfh_analysis/data_requests/2023_mayor_3-1 (branch: mayor_3-1)
added by scann0n